src/dom-updates.js

```diff
@@ -17,9 +18,13 @@ const innerModal = document.querySelector('.inner-modal');
 const tripMessage = document.querySelector('.trip-message');
 const vipStatus = document.querySelector('.vip-status');
 const navBtns = document.querySelectorAll('.nav-tab');
+const welcomeMessage = document.querySelector('.welcome-message');
+const coverImg = document.querySelector('.cover');
+const banner = document.querySelector('.banner');
```

Comment:
I see a lot of different selectors shared between dom-updates and scripts - any reason you decided to select them in both places rather than using the export/import?

Reply:
I didn't realize you could do that. I had tried with another let global variable to have it change its value but it was not working so I didn't think it was possible. But now that I got this to work it, I'm thinking it might have had an issue with the API